test/other_tests/twoD_vs_oneD.cpp

```diff
@@ -12,25 +12,25 @@
 // local
 #include "helper.h"
 
+#define TEST_NUM 10000000
+
 constexpr double MB = 1024 * 1024;
-constexpr size_t MILLION = 1000000;
-constexpr size_t TEST_NUM = 10 * MILLION;
```

Comment:
I change this because we can't compare constexpr size_t to uint8_t

Reply:
? where is the comparison

Reply:
for loop

Reply:
linux is giving me warning